tests/src/test/scala/pureconfig/ConfigReaderFailuresPrettyPrintSuite.scala

```diff
@@ -83,23 +80,6 @@ class ConfigReaderFailuresPrettyPrintSuite extends BaseSuite {
           |  - Expected type OBJECT. Found STRING instead.""".stripMargin
   }
 
-  it should "print a message displaying relevant errors for coproduct derivation" in {
```

Comment:
Where did this test go?

Reply:
I forgot to reintroduce them. Added them as tests in the pureconfig-generic-base module.